kml/serdes.cpp

```diff
@@ -336,13 +328,13 @@ void SaveCategoryData(Writer & writer, CategoryData const & categoryData,
 
     // Use CDATA if we have special symbols in the name.
     writer << kIndent2 << "<name>";
-    SaveStringWithCDATA(writer, GetLocalizableString(categoryData.m_name, kDefaultLang));
+    SaveStringWithCDATA(writer, *GetDefaultLanguage(categoryData.m_name));
```

Comment:
IDK how we ensure that name always exists (we wrote an empty string before?), but probably better here:
```
if (auto name = GetDefaultLanguage(categoryData.m_name))
{
  writer << kIndent2 << "<name>";
  SaveStringWithCDATA(writer, *name);
  writer << "</name>\n";
}
```

Reply:
Maybe helpfull ? 
Only recently it was noticed that gpx tags without content were a problem for Garmin connect mobile.
More info here: https://help.locusmap.eu/topic/36201-changed-route-planner-data#comment-145775
I myself do not employ the Garmin connect app as I currently do not use any Garmin device either.

Reply:
Fixed